Framework/Kernel/src/DateAndTime.cpp

```diff
@@ -381,30 +375,9 @@ const DateAndTime &DateAndTime::defaultTime() {
  * @param str :: ISO8601 format string: "yyyy-mm-ddThh:mm:ss[Z+-]tz:tz"
  * @param displayLogs :: flag to indiciate if the logs should be displayed
  */
-void DateAndTime::setFromISO8601(const std::string &str, bool displayLogs) {
+void DateAndTime::setFromISO8601(const std::string &str) {
   // Make a copy
   std::string time = str;
-  // Some ARGUS files have an invalid date with a space instead of zero.
```

Comment:
Does is make sense to add a unit test, checking that this throws now?

Reply:
Agreed.